lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -1938,7 +2070,10 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             request @(ApiConstructTransaction n) ctx constructEndpoint Default payload
 
         -- Submit tx
-        void $ submitTx ctx sealedTx [ expectResponseCode HTTP.status500 ]
+        submittedTx <- submitTxWithWid ctx w (ApiSerialisedTransaction sealedTx)
+        verify submittedTx
+            [ expectResponseCode HTTP.status500
```

Comment:
I'd advocate also for fixing this here. Submitting unsigned transaction should present proper error from wallet API, not `500` from the node.

Reply:
sounds reasonable